src/main/scala/rocket/icache.scala

```diff
@@ -128,7 +128,7 @@ class ICache(latency: Int)(implicit p: Parameters) extends CoreModule()(p) with
       io.resp.bits.datablock := Mux1H(s1_tag_hit, s1_dout)
       io.resp.valid := s1_hit
     case 2 =>
-      val s2_hit = RegEnable(s1_hit, !stall)
+      val s2_hit = RegEnable(s1_hit, Bool(false), !stall)
```

Comment:
rocket expects to be reset for 3 clocks--this should definitely be written down; I think the expectation was that any reset strategy would do that, so it never happened.